jbehave-support-core/src/main/java/org/jbehavesupport/core/ssh/SshSteps.java

```diff
@@ -70,7 +70,7 @@ public void init() {
         logCache.clear();
     }
 
-    private String readLog(String systemQualifier, ZonedDateTime startTime, ZonedDateTime endTime) {
+    public String readLog(String systemQualifier, ZonedDateTime startTime, ZonedDateTime endTime) {
```

Comment:
I'm still not sure that this API should be public. We should discuss it again and probably vote.

Reply:
This Is a reason for separate PR :)